artifact-manifest/artifact-manifest-wordpress-image-signature.json

```diff
@@ -0,0 +1,27 @@
+{
+  "schemaVersion": 1,
+  "mediaType": "application/vnd.oci.artifact.manifest.v1+json",
+  "artifactType": "application/vnd.cncf.notary.v2+json",
+  "config": {
+    "mediaType": "application/vnd.cncf.notary.config.v2+json",
+    "digest": "sha256:b5b2b2c507a0944348e0303114d8d93aaaa081732b86451d9bce1f432a537bc7",
+    "size": 102
+  },
+  "blobs": [
+    {
+      "mediaType": "application/tar",
+      "digest": "sha256:9834876dcfb05cb167a5c24953eba58c4ac89b1adf57f28f2f9d09af107ee8f0",
+      "size": 32654
+    }
+  ],
+  "manifests": [
+    {
+      "mediaType": "application/vnd.oci.image.manifest.v1+json",
+      "digest": "sha256:3c3a4604a545cdc127456d94e421cd355bca5b528f4a9c1905b15da2eb4a4c6b",
```

Comment:
I realize this is a strawman, but it seems like kind of an interesting choice to use a digest related to CVE-2020-15157 for it? :eyes:

Why not use something like `sha256:c477cb8b2a98570d8187f9e8dd652608a6c8d39e9fc401b895f6528df8d7f5fc` which is an actual digest for an amd64 image of a recent build of `wordpress:5`?

Reply:
lol, I think I just hacked a digest, and I apparently just happened to pick a CVE. I'll update to Wordpress, although I'm impressed that someone actually drilled into that level of details :)

Reply:
Frankly, there's so much here that it's overwhelming to read/parse (high level idea _plus_ spec format changes _plus_ tooling workflow _plus_ protocol changes), so my brain zeroed in on the one thing I have intimate subject matter expertise on, and when it turned out this digest wasn't something ever published by the official images program, I googled it and found CVE references. :sweat_smile:

Reply:
updated